documentation/concepts/statescripts.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: documentation
+menu: statescripts
+subtitle: State Scripts
+---
+# State Scripts
+{% include teams.html %}
+
+Over the lifetime of a project, many database objects may be created and destroyed across many migrations which leaves behind a lengthy history of migrations that need to be applied in order to bring a new environment up to speed.
+
+Instead, you might wish to add a single, cumulative script that represents the state of your database after all of those migrations have been applied without disrupting existing environments.
+
+[Flyway Teams Edition](/try-flyway-teams-edition) gives you a way to achieve this using **State Scripts**.
+
+## How it works
+
+Once you decide it's time to add such a cumulative script, instead of making it a regular [versioned migration](/documentation/concepts/migrations#versioned-migrations) it should be a state script, which is prefixed with `S` instead of `V`.
```

Comment:
I think this can be simpler. We can say:

'State scripts are prefixed with 'S', followed by the version they represent.

For instance, the state script `S5` represents the resultant state of all versioned migrations up to and including `V5`.